language-support/scala/codegen-testing/src/main/scala/com/digitalasset/ledger/client/binding/encoding/ShowEncoding.scala

```diff
@@ -119,7 +119,16 @@ object ShowEncoding extends ShowEncoding {
 
     override def valueOptional[A: Show]: Show[P.Optional[A]] = optionShow
 
-    override def valueMap[A: Show]: Show[P.Map[A]] = mapShow
+    override def valueTextMap[A: Show]: Show[P.TextMap[A]] = mapShow
+
+    override def valueGenMap[K: Show, V: Show]: Show[P.GenMap[K, V]] =
```

Comment:
This instance is worth defining implicitly in `InsertOrdMap`'s companion.

Reply:
done.